test/test_command.py

```diff
@@ -258,39 +258,83 @@ def setUp(self):
 
     @patch('subprocess.check_output')
     @patch('subprocess.check_call')
-    def test_remove_extra_packages_no_discrepancies(self, mock_check_call, mock_check_output):
+    def test_determine_extra_packages_no_discrepancies(self, mock_check_call, mock_check_output):
         mock_check_output.return_value = 'mock==1.2\nDjango==1.7\nnose==1.3\n'
         packages = _create_packages()
 
-        self.command.remove_extra_packages(packages)
+        self.assertEqual((), self.command.determine_extra_packages(packages))
```

Comment:
Ah.  I'd noticed that you did it sometimes one way, sometimes the other, but hadn't deduced the rule.  Are they OK now?